net/syncthing/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/syncthing/syncthing 1.0.0 v
+name                syncthing
```

Comment:
The `name` line can be removed, the github PortGroup (called by golang 1.0) will set it to `syncthing`.